app/controllers/articles_controller.rb

```diff
@@ -23,7 +23,7 @@ def new
   end
 
   def create
-    @article = Article.new(article_params)
+    @article = current_user.articles.build(article_params)
```

Comment:
Just added testing for all of the controller actions. Please check latest commit